uaclient/entitlements/fips.py

```diff
@@ -561,3 +566,19 @@ def _perform_enable(self, silent: bool = False) -> bool:
             return True
 
         return False
+
+
+class FIPSPreviewEntitlement(FIPSEntitlement):
+    name = "fips-preview"
+    title = messages.FIPS_PREVIEW_TITLE
+    description = messages.FIPS_PREVIEW_DESCRIPTION
+    help_text = messages.FIPS_PREVIEW_HELP_TEXT
+    pre_enable_msg = messages.PROMPT_FIPS_PREVIEW_PRE_ENABLE
+
+    @property
+    def incompatible_services(self) -> Tuple[IncompatibleService, ...]:
+        return super().incompatible_services + (
+            IncompatibleService(
+                FIPSEntitlement, messages.FIPS_INVALIDATES_FIPS_UPDATES
+            ),
+        )
```

Comment:
should we include livepatch and fips-updates in this list?

Reply:
Since we are inheriting from `FIPSEntitlement`, both of those services should already be covered. However, I will add this check directly on the integration test we are performing for the service

Reply:
Oh my eyes glazed over the `super()` line, sorry about that